e2e/apps-section.e2e.ts

```diff
@@ -50,21 +57,92 @@ describe('Attachment list', () => {
 
-        await apps.importPublishDeployApp(this.alfrescoJsApi, app.file_location);
+        firstApp = await apps.importPublishDeployApp(this.alfrescoJsApi, app.file_location);
+        appVersionToBeDeleted = await apps.importPublishDeployApp(this.alfrescoJsApi, appTobeDeleted.file_location);
 
         loginPage.loginToProcessServicesUsingUserModel(user);
 
         done();
     });
 
-    it('[C260198] Publish on ADF side', () => {
+    it('[C260198] Should the app be displayed on dashboard when is deployed on APS', () => {
         navigationBarPage.clickProcessServicesButton();
 
         processServicesPage.checkApsContainer();
 
-        expect(processServicesPage.getAppIconType(app.title)).toEqual('ac_unit');
+        expect(processServicesPage.getAppIconType(app.title)).toEqual(CONSTANTS.APP_ICON.UNIT);
         expect(processServicesPage.getBackgroundColor(app.title)).toEqual(CONSTANTS.APP_COLOR.BLUE);
-        expect(processServicesPage.getDescription(app.title)).toEqual('Description for app');
+        expect(processServicesPage.getDescription(app.title)).toEqual(app.description);
+    });
+
+    it('[C260213] Should a new version of the app be displayed on dashboard when is replaced by importing another app in APS', async() => {
+        navigationBarPage.clickProcessServicesButton();
+
+        processServicesPage.checkApsContainer();
+
+        expect(processServicesPage.getAppIconType(app.title)).toEqual(CONSTANTS.APP_ICON.UNIT);
+        expect(processServicesPage.getBackgroundColor(app.title)).toEqual(CONSTANTS.APP_COLOR.BLUE);
+        expect(processServicesPage.getDescription(app.title)).toEqual(app.description);
+
+        browser.controlFlow().execute(() => {
+            return apps.importNewVersionAppDefinitionPublishDeployApp(this.alfrescoJsApi, replacingApp.file_location, firstApp.id);
+        });
+
+        Util.refreshBrowser();
+
+        processServicesPage.checkApsContainer();
+
+        expect(processServicesPage.getAppIconType(app.title)).toEqual(CONSTANTS.APP_ICON.FAVORITE);
+        expect(processServicesPage.getBackgroundColor(app.title)).toEqual(CONSTANTS.APP_COLOR.GREY);
+        expect(processServicesPage.getDescription(app.title)).toEqual(app.description);
+    });
+
+    it('[C260220] Should the app not be displayed on dashboard after it was deleted in APS', async() => {
+        navigationBarPage.clickProcessServicesButton();
+
+        processServicesPage.checkApsContainer();
+
+        processServicesPage.checkAppIsDisplayed(app.title);
+
+        browser.controlFlow().execute(() => {
+            return modelActions.deleteEntireModel(this.alfrescoJsApi, firstApp.id);
+        });
+
+        Util.refreshBrowser();
+
+        processServicesPage.checkApsContainer();
+        processServicesPage.checkAppIsNotDisplayed(app.title);
+    });
+
+    it('[C260220] Should the penultimate version of an app be displayed on dashboard when the last version is deleted in APS', async() => {
```

Comment:
the number of this test should be C260215 , can you also change the flag on testrail fo the field Automated Execution?